server/data/hmppsAuthClient.ts

```diff
@@ -48,14 +55,14 @@ export default class HmppsAuthClient {
   }
 
   getUser(token: string): Promise<User> {
-    logger.info(`Getting user details: calling HMPPS Auth`)
-    return HmppsAuthClient.restClient(token).get({ path: '/api/user/me' }) as Promise<User>
+    logger.info('Getting user details: calling HMPPS Auth')
+    return HmppsAuthClient.restClient(token).get<User>({ path: '/api/user/me' })
```

Comment:
Maybe a separate ticket, but we should be pointing to Manage Users API for /user/me(/roles) etc.

Reply:
(See the work that Ravi's done recently)

Reply:
Yep - perhaps we should be making a PR to the template app for that one